integration/examples/custom-tests/skaffold.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: skaffold/v2beta13
+kind: Config
+build:
+  artifacts:
+  - image: skaffold-example
+test:
+  - image: skaffold-example
+    custom:
+      - command: ./test.sh
+        timeoutSeconds: 60
+        dependencies:
+          paths:
+          -  "*_test.go"
+      - command: echo Hello world!!
```

Comment:
Could you add:
```
dependencies:
  command: echo "[main_test.go]"
```
Then we can test both styles of specifying test dependencies.

Reply:
Added the suggested dependency.